db/migrate/20190314010045_drop_users.rb

```diff
@@ -0,0 +1,5 @@
+class DropUsers < ActiveRecord::Migration[5.2]
```

Comment:
The migration history will be cleaner if you don't drop this table.  Instead, delete the other migration file (https://github.com/r3drighthand/social-animals/blob/master/db/migrate/20190309210309_add_display_name_to_users.rb), drop the database with `db:drop`, and then run `db:create` and `db:migrate` to update `schema.rb`.  This isn't necessary but since we are still at the very beginning, let's keep it as clean as we can.

Reply:
Good idea.